pages/stable-version.html

```diff
@@ -131,7 +131,7 @@ <h5>Hardware and Software Requirements</h5>
                     <tbody >
                     <tr>
 
-                        <td>openSUSE Leap 15.4</td>
```

Comment:
Please remember to adjust this, and other similar things at `pages/devel-version.html` as well (master)

Reply:
Changed 15.4 to 15.5 in 3 different places of that file.